### PR TITLE
Remove osql block cbacks

### PR DESCRIPTION
**Pull request by dorinhogea**
These are unused, blocksql related callbacks in NET. Code removed.

### CODE REVIEW

File: db/osqlcomm.c [hunk 1]
```diff
@@ -5061,13 +5004,9 @@ void *osql_create_request(const char *sql, int sqlen, int type,
     }
 
     p_buf = buf_no_net_put(sql, sqlen, p_buf, p_buf_end);
-    if (type == OSQL_BLOCK_REQ_COST)
+    if (tag) {
```
Comment (adizaimi) [inline db/osqlcomm.c]: merge with the next if below?

File: db/comdb2.h [hunk 2]
```diff
@@ -1320,11 +1320,7 @@ struct ireq {
     /* copy of blkseq */
     uint8_t seq[MAX_SEQ_LEN];
 
-    /* for blocksql purposely we want to have the list of sql
-       queries submitted;
-       the field is not null if we have any BLOCK2_SQL requests
-       THIS IS USED IN BLOCKSQL, WOULD BE NICE TO UNIFY -later-
-     */
+    /* bplog pointer */
     void *blocksql_tran;
```
Comment (adizaimi) [inline db/comdb2.h]: can we rename this for clarity?

Comment (dorinhogea) [reply db/comdb2.h]: there is a bigger rework of this coming up

File: db/osqlcomm.c (hunk 1 above)
Comment (dorinhogea) [reply db/osqlcomm.c]: sure

**Comment by dorinhogea** [comment]
thanks for review

**Pull request merged**